_includes/desktop-install.html

```diff
@@ -10,10 +10,10 @@ <h5 class="panel-title">
       </div>
       <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
         <div class="panel-body">
-          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/96739/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a>
         <br>
         <br>
-        <b>Checksum:</b> SHA-256 69ea659b0ca0e160a1de9bd63dc5697f5eb89fff1d33484fb8ef9793e43d0d45
+        <b>Checksum:</b> SHA-256 5f6db3cf5a2084fc7c584c90792f38a0caac91c4eed4f8653dde7bb8148517f1
```

Comment:
Interesting; looks like for the other locations where we linked to `main`, we mentioned a link to the release notes instead of providing a checksum (perhaps something we should consider doing here as well to prevent having to update multiple locations?)

Can be a follow up though